android-exoplayer/src/main/java/com/brentvatne/exoplayer/ReactExoplayerView.java

```diff
@@ -326,6 +338,33 @@ private void addPlayerControl() {
         addView(playerControlView, 1, layoutParams);
     }
 
+    /**
+     * Update fullscreen icon
+     */
+    private void updateFullScreenIcon(Boolean fullScreen) {
+        if(playerControlView != null && player != null) {
+            //Play the video whenever the user clicks minimize or maximise button. In order to enable the controls
+            player.setPlayWhenReady(true);
+            ImageView fullScreenIcon = playerControlView.findViewById(R.id.exo_fullscreen_icon);
```

Comment:
I don't think this should be part of react-native-video. Each video application can implement this on the javascript side with the value of fullscreen prop

Reply:
@benoitdion
This is in continuation with the already merged feature of showing media controls in ExoPlayer which was not available earlier. 
Supporting full-screen control in the same ExoPlayer is requested by many RN community followers.